cmd/spire-server/cli/agent/show.go

```diff
@@ -62,6 +62,11 @@ func (c *showCommand) Run(ctx context.Context, env *common_cli.Env, serverClient
 		return err
 	}
 
+	if agent.Selectors != nil {
+		for _, s := range agent.Selectors {
```

Comment:
Can you move this into `printAgents`? Can you also align the `:` for each line being printed?

Reply:
`printAgents` is also used by `spire-server agent list` command.
I think this feature should be only activate in `spire-server agent show` command.

Reply:
Aligned to other items. https://github.com/spiffe/spire/pull/2133/commits/9dc0016b55a5a885cf25227a4dc1ba3c0e50b9ed

Reply:
> `printAgents` is also used by...
Oh yep. You are right. Thanks!